source/rpc/client.py

```diff
@@ -52,25 +57,29 @@ def imagerProxy(self) -> ImagerProxy:
     @property
     def session(self) -> Session:
         if self.sessionProxy:
-            return getattr(self, "_session")
+            return Session(sessionProxy=self.sessionProxy, device=self)
+            # return getattr(self, "_session")
```

Comment:
More commended code. + 67, 76, 82.

Reply:
fixed in e92e4df4543e7958c7dcf211f92bf0dc11da51c4

Reply:
I can see it on this branch head.

Reply:
Sry. Fixed in 3f24e51d3b9f59dcded7f01f51dc5bec441c1c4c